.github/workflows/rust.yml

```
@@ -10,27 +10,21 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build:
+  cargo:
     runs-on: ubuntu-latest
-
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - command: clippy -- -D warnings
+          - command: fmt -- --check
+          - command: test --locked
     steps:
-    - uses: actions/checkout@v4
-    
-    - name: Setup Rust
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        override: true
 
-    
-    - name: Build
-      run: cargo build --verbose
+    - uses: actions/checkout@v3
 
-    - name: Run tests
-      run: cargo test --locked --verbose
+    - uses: actions-rs/toolchain@v1
```

Hmmmmm looks like it's failing on this action.

The error says no `rust-toolchain` file is present in the repo, but this shouldn't be needed.

What if you change this line too:

```suggestion
    - uses: actions-rs/toolchain@v1
      with:
        toolchain: stable
```

This just specifies the `stable` toolchain, though tbh I thought this was the default :thinking: